torchbenchmark/models/pyhpc_isoneutral_mixing/metadata.yaml

```
@@ -1,3 +1,6 @@
+devices:
+  NVIDIA A100-SXM4-40GB:
+    eval_batch_size: 67108864
```

```suggestion
    eval_batch_size: 1048576
```
In our sheet, its TFLOPS is only 0.01 and I think we can choose the first 0.01 to save execution time.